base/Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN yes | unminimize \
 
 ENV LANG=en_US.UTF-8
 
+ENV TRIGGER_REBUILD=1
```

Comment:
For what it's worth, this line change looks good to me, and should work as intended after the PR is merged and deployed.

(Didn't look too closely at the other changes.)